<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserRegisterDialog</class>
 <widget class="QDialog" name="UserRegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>24</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>55</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>124</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leEmail">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>23</y>
     <width>166</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePwd">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>54</y>
     <width>166</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>至少3位以上</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNick">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>85</y>
     <width>166</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>86</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOk_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>81</y>
     <width>65</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>随机</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
